seatunnel-e2e/seatunnel-connector-v2-e2e/connector-tdengine-e2e/src/test/resources/tdengine/tdengine_source_to_sink.conf

```
@@ -0,0 +1,99 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######
+###### This config file is a demonstration of streaming processing in seatunnel config
+######
+
+env {
+  # You can set flink configuration here
+  execution.parallelism = 2
+  job.mode = "STREAMING"
+  #execution.checkpoint.interval = 10000
+  #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
+}
+
+source {
+  # This is a example source plugin **only for test and demonstrate the feature source plugin**
+#   FakeSource {
+#     parallelism = 2
+#     result_table_name = "fake"
+#     row.num = 16
+#     schema = {
+#       fields {
+#         name = "string"
+#         age = "int"
+#       }
+#     }
+#   }
+    TDengine {
+      url : "jdbc:TAOS-RS://flink_e2e_tdengine_src:6041/"
+      username : "root"
+      password : "taosdata"
+      database : "power"
+      stable : "meters"
+      lower_bound : "2018-10-03 14:38:05.000"
+      upper_bound : "2018-10-03 14:38:16.801"
+      fields {
+        ts = "timestamp"
+        current = "float"
+        voltage = "int"
+        phase = "float"
+      }
+      tags {
+        location = "string"
+        groupid = "int"
+      }
+      result_table_name = "tdengine_result"
+    }
+  # If you would like to get more information about how to configure seatunnel and see full list of source plugins,
+  # please go to https://seatunnel.apache.org/docs/category/source-v2
+}
+
+transform {
+  sql {
+    sql = "select * from tdengine_result"
+  }
+
+  # If you would like to get more information about how to configure seatunnel and see full list of transform plugins,
+  # please go to https://seatunnel.apache.org/docs/category/transform
+}
```

sql transform  has been removed

```suggestion
transform {
}
```